tinygrad/runtime/support/elf.py

```diff
@@ -23,7 +23,7 @@ def _to_carray(sh, ctype): return (ctype * (sh.header.sh_size // sh.header.sh_en
   # Prealloc image for all fixed addresses.
   image = bytearray(max([sh.header.sh_addr + sh.header.sh_size for sh in progbits if sh.header.sh_addr != 0] + [0]))
   for sh in progbits:
-    if sh.header.sh_addr != 0: image[sh.header.sh_addr:sh.header.sh_addr+sh.header.sh_size] = sh.content
+    if sh.header.sh_addr != 0 or sh.name == '.text': image[sh.header.sh_addr:sh.header.sh_addr+sh.header.sh_size] = sh.content
```

Comment:
Why does 0 need a special rule in the first place?

Reply:
If the section does not appear in the process' memory then `sh_addr` is set to 0 (see `man 5 elf`). I think this is relevant for the NV backend?